src/WSRequestHandler_SceneCollections.cpp

```diff
@@ -12,26 +17,32 @@
  * @category scene collections
  * @since 4.0.0
  */
-RpcResponse WSRequestHandler::SetCurrentSceneCollection(const RpcRequest& request) {
-	if (!request.hasField("sc-name")) {
+RpcResponse WSRequestHandler::SetCurrentSceneCollection(const RpcRequest &request)
+{
+	if (!request.hasField("sc-name"))
+	{
 		return request.failed("missing request parameters");
 	}
 
-	const char* sceneCollection = obs_data_get_string(request.parameters(), "sc-name");
-	if (!sceneCollection) {
+	const char *sceneCollection = obs_data_get_string(request.parameters(), "sc-name");
+	if (!sceneCollection)
+	{
 		return request.failed("invalid request parameters");
 	}
 
-	char** collections = obs_frontend_get_scene_collections();
+	char **collections = obs_frontend_get_scene_collections();
 	bool collectionExists = Utils::StringInStringList(collections, sceneCollection);
 	bfree(collections);
-	if (!collectionExists) {
+	if (!collectionExists)
+	{
 		return request.failed("scene collection does not exist");
 	}
 
-	obs_queue_task(OBS_TASK_UI, [](void* param) {
-		obs_frontend_set_current_scene_collection(reinterpret_cast<const char*>(param));
-	}, (void*)sceneCollection, true);
+	obs_queue_task(
+		OBS_TASK_UI, [](void *param) {
+			obs_frontend_set_current_scene_collection(reinterpret_cast<const char *>(param));
+		},
+		(void *)sceneCollection, true);
```

Comment:
Why did it end up like this?

Reply:
Auto-formatting, my fault. Actually there are no relevant changes there, so I reverted that from my last commit.